Source: Gentoo
Upstream: Yes.
Reason: Fix library name and, incidentally, make it usable with --as-needed

diff --git a/saslauthd/configure.in b/saslauthd/configure.in
index cd37b4c..4dd4fd1 100644
--- a/saslauthd/configure.in
+++ b/saslauthd/configure.in
@@ -77,7 +77,7 @@ if test "$authsasldb" != no; then
   AC_DEFINE(AUTH_SASLDB,[],[Include SASLdb Support])
   SASL_DB_PATH_CHECK()
   SASL_DB_CHECK()
-  SASL_DB_LIB="$SASL_DB_LIB ../sasldb/.libs/libsasldb.al"
+  SASL_DB_LIB="../sasldb/.libs/libsasldb.a $SASL_DB_LIB"
 fi

 AC_ARG_ENABLE(httpform, [  --enable-httpform       enable HTTP form authentication [[no]] ],
